MarsQA-CompetitionTask2/Test/Certificate_Test.cs

```diff
@@ -0,0 +1,203 @@
+﻿using AventStack.ExtentReports.Reporter;
+using AventStack.ExtentReports;
+using MarsQA_CompetitionTask2.Pages.Skill;
+using MarsQA_CompetitionTask2.Utilits;
+using NUnit.Framework;
+using NUnit.Framework.Internal.Commands;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MarsQA_CompetitionTask2.Data;
+
+namespace MarsQA_CompetitionTask2.Test
+{
+    public class Certificate_Test : CommonDriver
+    {
+        Login loginobj;
+        Profile profileobj;
+        Edit editObj;
+        Delete deleteObj;
+        CertificationData CertificationData;
+        public static ExtentReports extent;
+        public static ExtentTest test;
+
+
+
+        public Certificate_Test()
+        {
+            loginobj = new Login();
+            profileobj = new Profile();
+            editObj = new Edit();
+            deleteObj = new Delete();
+
+
+        }
+        [OneTimeSetUp]
+        public static void ExtentStart()
```

Comment:
so something minor but it is best practice to put your oneTimeSetup in a hooks class as it executes once which is used to setup things one time for example your extent report should be declared once not in every class and then you should use the instance in classes https://docs.nunit.org/articles/nunit/writing-tests/attributes/onetimesetup.html

for your advance task please make sure you fix this

Reply:
Done Changes .Please review my code now.